Visualizer/src/components/dashboard/AgentInfo.vue

```diff
@@ -251,7 +391,12 @@ const allocations = computed(() => {
       },
       {
         label: "Collisions",
```

Comment:
Collisions is confusing: no collision happened. Those are number of agents that had to reallocate because of the selected agent: Nr. Reallocations Forced?